saleor/payment/gateways/braintree/__init__.py

```diff
@@ -159,7 +159,7 @@ def authorize(
             last_4=credit_card.get("last_4"),
             exp_year=credit_card.get("expiration_year"),
             exp_month=credit_card.get("expiration_month"),
-            brand=credit_card.get("card_type", "").lower(),
+            brand=credit_card.get("card_type", "").lower() if credit_card.get("card_type", "") else "",
```

Comment:
It would be cleaner to use a variable and not duplicate the `get()` call:
```
brand = credit_card.get("card_type", "")
brand = brand.lower() if brand is not None else ""
```
And later use it in the constructor of PaymentMethodInfo.
Also, this line causes CI to fail as it is too long. The code needs to be reformatted with Black.

Reply:
@meska Would you have time to apply the changes?

Reply:
Yes, I will do another pull request with your suggested changes